SocialCareCaseViewerApi/V1/Boundary/Response/CaseSubmissionResponse.cs

```diff
@@ -35,6 +35,10 @@ public class CaseSubmissionResponse
 
         public int? CompletedSteps { get; set; }
 
+        public bool Deleted { get; set; } = false;
+        public DeletionDetails? DeletionDetails { get; set; }
+        public bool IsImported { get; set; } = false;
```

Comment:
this is never set?

Reply:
I'm not a big fan of adding that "dummy" IsImported property here, but it's there to inform the front end what records can be deleted (we shouldn't touch imported records). The way the front end combines records from submissions and cases end points makes this a bit awkward, but since we are not going to support cases deletion for now, this placeholder property should help the front end with business logic. At least it's not polluting the records themselves and can be removed or develop further if need be.